fsspec/asyn.py

```diff
@@ -151,6 +151,18 @@ def get_loop():
     return loop[0]
 
 
+def reset_after_fork():
+    global lock
+    loop[0] = None
+    iothread[0] = None
+    lock = None
```

Comment:
Are you sure this works? Why is only the `lock` declared as `global`. Furthermore, there is no `lock` in this file, only `_lock` and `get_lock`. This got changed in ffe57d6eabe517b4c39c27487fc45b804d314b58. Furthermore, this function is basically identical to `reset_lock`, so why not use that function?